app/src/main/java/org/systers/mentorship/view/activities/AboutActivity.kt

```diff
@@ -0,0 +1,38 @@
+package org.systers.mentorship.view.activities
+
+import android.content.Intent
+import android.content.res.Resources
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_about.*
+import org.systers.mentorship.R
+
+class AboutActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_about)
+
+        btnGit.setOnClickListener {
+            val string = R.string.url_github
+            startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(string.toString())))
+        }
+        btnSlack.setOnClickListener {
+            startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(R.string.url_slack.toString())))
```

Comment:
```suggestion
            startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.url_slack))))
```

Reply:
When I tested this out on an emulator, I noticed that the app breaks when I click a link in the About page. This is because in this line of code (and the following), when you give the url to be parsed, you're actually giving an integer in the string format.
So instead of calling `toString()` you should call a method such as `getString()` to actually tell the activity, to convert the resource `R.string.url_slack` into the string value it represents `systers-opensource ....` @SaumyaSingh1

Reply:
this line of code wasn't changed, this will cause an exception most likely. Can you please change it to be:
```
startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.url_slack))))
```